### PR TITLE
fix: user info provider complete. merge remainings

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

**Comment by bladicito** [comment]
@lehmi11, after merging some stuff was missing (post replies and others). I just corrected it.

File: services/Mumble/MumblePost.ts [hunk 1]
```diff
@@ -127,7 +123,7 @@ export class MumblePostService extends MumbleService {
         accept: 'application/json',
       },
     });
-    return responseApi as IPostItem;
+    return responseApi as any;
```
Comment (lehmi11) [inline services/Mumble/MumblePost.ts:126]: Is that really any or can we use IPotstItem?

**Review by lehmi11 — APPROVED** [review]
Only a comment about an any-type in code. Otherwise looks good

File: services/Mumble/MumblePost.ts (hunk 1 above)
Comment (bladicito) [reply services/Mumble/MumblePost.ts:126]: no we get a raw item (without createdDateTime). 
Will type it. 👍🏽

**Pull request merged**